src/goDocumentation.ts

```diff
@@ -152,12 +170,49 @@ function _packageNotFound(stderr: string): boolean {
 	return !!match;
 }
 
-function _cleanupGodocOutput(output: string): string {
+function _cleanupGodocOutput(output: string, html: boolean): string {
+	return html ? _cleanupHtmlGodocOutput(output) : _cleanupTextGodocOutput(output);
+}
+
+function _cleanupTextGodocOutput(output: string): string {
 	output = output.replace(/.*\n\n(PACKAGE DOCUMENTATION\n\n.*)/, '$1');
 	output = output.replace(/import ".*\/vendor\/(.*?)"/, 'import "$1"');
 	return output;
 }
 
+function _cleanupHtmlGodocOutput(output: string): string {
+	let staticPath = godocHtmlStaticPath();
+
+	output = output.replace(/document\.ANALYSIS_DATA = ;/, '');
+	output = output.replace(/document\.CALLGRAPH = ;/, '');
+
+	output = `
+		<!DOCTYPE html>
+		<html>
+		<head>
+			<!--temporarily commented until we figure out what the licence situation is
```

Comment:
@ramya-rao-a so this bit is pending the answer to https://github.com/Microsoft/vscode-go/issues/789#issuecomment-338488571

Reply:
Otherwise this will take some CSS + JS wizardry to make the output pretty.